app/dashboard/templates/shared/nav_chat.html

```diff
@@ -0,0 +1,6 @@
+<div id="gc-chat" class="nav-item order-md-last">
+  <a href="/chat/" class="chat_icon nav-link" role="button">
+    <i class="fas fa-comments fa-fw"></i>
+    <span id="notification-dot" class="notification__dot"></span>
```

Comment:
is this dot being used here?

Reply:
It will be, the idea is to have it show any notifications from the chat server, right now it should just show to indicate online, it doesn't do anything besides aesthetics right now.